app/helpers/layout_helper.rb

```diff
@@ -81,7 +81,7 @@ def page_entries_info(collection, options = {})
   end
 
   def will_paginate_with_info(collection = nil, options = {})
-    content_tag(:div, :id => "pagination", :class => "row") do
+    content_tag(:div, :id => "pagination", :class => "row", 'data-meta-count' => collection.total_entries) do
```

Comment:
I've removed the use of meta attribute, instead I've expand  `will_paginate_with_info` with `include_data_tag` flag.